front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorForm.tsx

```diff
@@ -65,6 +69,7 @@ const RollingStockEditorForm = ({
   const { t } = useTranslation();
   const { t: rollingStockT } = useTranslation('translation', { keyPrefix: 'rollingStock' });
   const { openModal } = useModal();
+  const { closeModal } = useContext(ModalContext);
```

Comment:
One last comment, you can get closeModal from useModal ;)
```suggestion
  const { openModal, closeModal } = useModal();
```